### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Nessrine88**
### Project requirements

- Add any missing JSX elements to the project 
- The final design match Zeplin's design 
-  Using the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/style/Book.css [hunk 1]
```diff
@@ -1,17 +1,135 @@
...
+.chapter {
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1.5rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: var(--black-two);
+}
```
Comment (Munsa1) [inline src/style/Book.css]: - [ ] Please use reusable classes for reoccurring CSS styles. For example consider grouping the following styles then using that class on all the desired components:



```
### Example
.reusable-class{

  font-family: 'RobotoSlab', sans-serif;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
}
```

### Note that this should be applied to `navigation.css` file as well

File: src/style/Navigation.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+ul {
+  display: flex;
```
Comment (Munsa1) [inline src/style/Navigation.css]: - [ ] The same here, `display: flex` is used three times in this file, using a reusable class helps make our code DRY

File: src/style/Book.css [hunk 3]
```diff
@@ -1,17 +1,135 @@
...
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
```
Comment (Munsa1) [inline src/style/Book.css:31]: - [ ] The same thing here consider using a reusable class for this `color: var(--black-two);` as you have used it 4 times in this file

File: src/style/BookForm.css [hunk 4]
```diff
@@ -9,13 +9,42 @@ form input {
   border: 1px solid #ccc;
```
Comment (Munsa1) [inline src/style/BookForm.css:9]: - [ ] The same thing applies to this border style, it is being used accross multiple CSS files, the DRY principle means that we apply it in one file as a reusable class then simply import that CSS file into all the affected components, than having it repeat in each CSS file

File: src/components/Navigation.js [hunk 5]
```diff
@@ -1,59 +1,33 @@
...
+      </div>
+      <div className="list">
+        <div className="ul">
+          <ul>
+            <li>
+              <Link to="/">Home</Link>
+            </li>
+            <li>
+              <Link to="/categories">Categories</Link>
+            </li>
```
Comment (Munsa1) [inline src/components/Navigation.js:22]: - [ ] Kindly follow the given design in terms of styling for the following part of the app\

Your design
image

Given design
image

File: src/components/Book.js [hunk 6]
```diff
@@ -12,16 +13,33 @@ function Book({
...
+        <br />
+        <span className="state">Completed</span>
       </p>
-      <button type="button" onClick={handleDelete}>
-        Remove
-      </button>
+      <div className="last">
+        <p className="comment">Current Chapter</p>
+        <p className="chapter">Chapter 17</p>
+        <button type="button" className="btn">UPDATE PROGRESS</button>
```
Comment (Munsa1) [inline src/components/Book.js:41]: - [x] In the given design this text uses the `RobotoSlab-Light` font while yours is using the Arial font, kindly fix this


Note that this applies for the `Add Book` button as well
image

File: src/components/Book.js [hunk 7]
```diff
@@ -12,16 +13,33 @@ function Book({
...
+        60%
+        <br />
+        <span className="state">Completed</span>
       </p>
-      <button type="button" onClick={handleDelete}>
-        Remove
-      </button>
+      <div className="last">
+        <p className="comment">Current Chapter</p>
+        <p className="chapter">Chapter 17</p>
```
Comment (Munsa1) [inline src/components/Book.js]: - [ ] Kindly use the correct font for the `Chapter` text as well so as to match the given design

Your design
image

Given design
image

File: src/components/BookForm.js [hunk 8]
```diff
@@ -31,33 +31,28 @@ function BookForm() {
...
           value={category}
           onChange={(e) => setCategory(e.target.value)}
-        />
-        <button type="submit">
+        >
+          <option value="">Category</option>
+          <option value="option1">Fiction</option>
+          <option value="option2">Action</option>
+          <option value="option3">Romance</option>
+        </select>
```
Comment (Munsa1) [inline src/components/BookForm.js]: - [ ] Kindly consider adding the correct font to the input placholder text and to the drop down selection inorder to match the given design

Your design
image

Given design
image

Note that your dropdown menu is grey in color while the given one is white

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Nessrine88, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Followed Given Design :heavy_check_mark: 
Good UI Design :heavy_check_mark: 
No Inline Styles :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KDlamini — APPROVED** [review]
### Hi @Nessrine88 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Professional PR title and description
✔️ Styling matches the given Zeplin design
✔️ The app makes use of basic CSS best practices

# Status: Approved ✔️ 

LeonardoInGatsby

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**